README.md

```diff
@@ -167,10 +167,20 @@ $ ninja -C build && ( cd build ; lit test )
 - `TRITON_INTERPRET=1` uses the Triton interpreter instead of running on the
   GPU.  You can insert Python breakpoints in your kernel code!
 - `TRITON_ENABLE_LLVM_DEBUG=1` passes `-debug` to LLVM, printing a lot of
-  debugging information to stdout.  If this is too noisy, run with just
+  debugging information to stdout. If this is too noisy, please try
+  `TRITON_LLVM_DEBUG_ONLY` instead to limit the output.
+
+  The other alternative to reduce output noisiness is running with just
```

Comment:
Instead of "The other alternative to reduce", say "An alternative way to reduce"